zerver/webhooks/stripe/view.py

```diff
@@ -1,7 +1,7 @@
 # Webhooks for external integrations.
 import time
 from datetime import datetime
-from typing import Any, Dict, List, Optional, Tuple
+from typing import Any, Dict, List, Optional, Tuple, Union
```

Comment:
Importing `Union` is not necessary here. :)

Reply:
Actually, I was trying something so I included it there but forgot to remove it before pushing.